sql_runner/snowflake_target.go

```diff
@@ -80,32 +81,42 @@ func (sft SnowFlakeTarget) RunQuery(query ReadyQuery, dryRun bool, showQueryOutp
 		return QueryStatus{query, query.Path, 0, nil}
 	}
 
-	scripts := strings.Split(query.Script, ";")
+	// The MULTI_STATEMENT_COUNT parameter (through WithMultiStatement)
```

Comment:
I think this comment can be much shorter eg

`// 0 allows arbitrary number of statements`